.appveyor.yml

```diff
@@ -20,6 +24,8 @@ clone_folder: *REPO_DIR
 cache:
   - *CACHE_DIR
 
+clone_depth: 50
```

Comment:
I can't recall exactly but there is this change: f12b3002982edd0e4db05034f931095db242129e